packages/gatsby/src/state-machines/data-layer/index.ts

```diff
@@ -48,15 +48,6 @@ const initialCreatePagesStates: StatesConfig<IDataLayerContext, any, any> = {
     invoke: {
       id: `creating-pages`,
       src: `createPages`,
-      onDone: {
-        target: `rebuildingSchemaWithSitePage`,
-        actions: `assignChangedPages`,
```

Comment:
This drops `assignChangedPages` action in `onDone` - is it intentional? It does look wrong at first glance

Reply:
You are right, restored this action. Looks like we don't have tests for it as it wasn't triggering any failures